docs/source/settingup.rst

```diff
@@ -151,6 +151,42 @@ Once the dataset is available on Dataverse, it can also be cloned using the
       'datalad-annex::?type=external&externaltype=dataverse&encryption=none&exporttree=no&url=https%3A//demo.dataverse.org&doi=doi:10.70122/MYT/ESTDOI'
 
 
+Help others getting started
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Its not always obvious to outsiders whether a Dataverse Dataset is a DataLad Dataset as well.
+To help others get started with your shared datasets, we recommend to add a short DataLad-specific description to your dataset's "Description" metadata.
+For convenience, a template to copy-and-paste can be found below.
+
+.. code-block:: text
+
+   This dataset is a <b>DataLad dataset</b> (https://www.datalad.org), published with
+   the <code>DataLad-dataverse</code> software (https://docs.datalad.org/projects/dataverse).
```

Comment:
```suggestion
   the datalad-dataverse software (https://docs.datalad.org/projects/dataverse).
```
I think `<code>` is not necessary, further down the software names are spelled without it.

Also further down, the extension name is all-lowercase. I'd suggest being consistent this way, but any other way would be fine too.